src/index.js

```diff
@@ -30,11 +30,27 @@ if (!isProduction) {
 }
 
 function run() {
-  const promises = meetups.map(meetup => getMeetupEventsData(`https://api.meetup.com/${meetup}/events`));
   const resolveAllPromises = isProduction ? resolveMeetupEventsDataS3 : resolveMeetupEventsDataLocal;
+  const eventsData = getMeeptupEventsData();
+  const groupsData = getMeeptupGroupsData();
+  let groupImages = {};
 
-  Promise.all(promises)
-    .then(resolveAllPromises)
+  Promise.all([eventsData, groupsData])
+    .then(function (results) {
+      results[0] = JSON.parse(formatResults(results[0]));
```

Comment:
I general I think it might be more reliable / clearer to operate an a variable "custom" to the processing we're doing here, rather than overwriting / re-assigning existing params.  

So something like
```
Promise.all([eventsData, groupsData])
  .then(function (results) {
    let events = JSON.parse(formatResults(results[0]));
    let groups = results[1];  // why don't we need formatResults() here though???
    let groupImages = {};
    
    // Build object containing group name, and associated group image.
    groups.map(group => {
      groupImages[group.urlname] = group.group_photo !== undefined ? group.group_photo.highres_link : null;
    });

    // Append this image to event
    events.map(event => {
      event.group.group_photo = group groupImages[event.group.urlname]; // eslint-disable-line
    });

    resolveAllPromises(events);
  })
  .catch(handleError);
```

Reply:
I used formatResults() to remove the empty objects in the events data. I think that it also was enclosed inside another array for some reason, and formatResults() removed that. The groups data came out fine though, which is why it didn't need it.